docs/en/latest/pubsub/kafka.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Apache Kafka
+keywords:
+  - APISIX
+  - Pub-Sub
+  - Kafka
+description: This document contains information about the Apache APISIX kafka pub-sub scenario.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Connect to Apache Kafka
+
+Connecting to Apache Kafka in Apache APISIX is very simple.
+
+Currently we provide a simpler way to integrate by combining two APIs, ListOffsets and Fetch, to quickly implement the ability to pull Kafka messages, but do not support Apache Kafka's consumer group feature for now, and cannot be managed by Kafka for offsets.
+
+### Limitations
+
+- Offsets need to be managed manually
+They can be stored by a custom backend service or obtained via the list_offset command before starting to fetch the message, which can use timestamp to get the starting offset, or to get the initial and end offsets.
+- Unsupported batch data acquisition
+A single instruction can only obtain the data of a Topic Partition, does not support batch data acquisition through a single instruction
+
+### Prepare
+
+First, it is necessary to compile the [communication protocol](../../../../apisix/pubsub.proto) as a language-specific SDK using the `protoc`, which provides the command and response definitions to connect to Kafka via APISIX using the WebSocket.
```

Comment:
Let's update the path of definition. We should use absolute path as it's not in website

Reply:
changed